controllers/AdController.php

```diff
@@ -1,123 +1,278 @@
 <?php
+//
```

Comment:
Next time it's better to delete unused code rather than comment. You have git, use it properly